packages/govuk-frontend-review/tasks/watch.mjs

```diff
@@ -38,7 +43,14 @@ export const watch = (options) =>
     task.name('compile:scss watch', () =>
       gulp.watch(
         ['**/*.scss', join(paths.package, 'dist/govuk/all.scss')],
-        { cwd: options.srcPath },
+        {
+          awaitWriteFinish: true,
+          cwd: options.srcPath,
+
+          // Prevent early Sass compile by ignoring delete (unlink) event
+          // when GOV.UK Frontend runs the `clean` script before build
+          events: ['add', 'change']
```

Comment:
**note** Thinking this will also skip `unlink` events for files matching `**/*.scss`. Brings a slim risk of not recompiling straigh after having mistakenly deleted a `.scss` file from the review app but not the corresponding `@import`, which would delay the realisation of that deletion. This would hopefully be caught at commit time (if adding file by file) or by CI so that skiping of all `unlink` seems our least worst option there. Shame the [`ignored` option](https://gulpjs.com/docs/en/api/watch/#options) doesn't get us enough info 😢